lib/tags/index.js

```diff
@@ -18,11 +18,14 @@ export default class TagContainer extends Component {
     const {
       tags,
       removeTag,
-      resetColour
+      resetColour,
+      resetBackgroundColour,
+      resetBorderColour
     } = this.props;
     const resetButtonStyle = {
       color: resetColour,
-      border: '1px solid' + resetColour
+      backgroundColor: resetBackgroundColour,
+      border: '1px solid' + resetBorderColour
     };
```

Comment:
it is because it changes position depending on whether you're on mobile or desktop